week-1/exercises/initials.js

```diff
@@ -1,6 +1,12 @@
 let firstName = "Creola";
 let middleName = "Katherine";
 let lastName = "Johnson";
+let initials = (firstName.substring(0,3)).toUpperCase()  + " " + (middleName.substring(0,3)).toUpperCase() + " "+ (lastName.substring(0,3)).toUpperCase() 
```

Comment:
Can you refactor this expression to make use of a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) ?